src/features/student/views/StudentEducationAmsView/StudentEducationAmsView.vue

```diff
@@ -1,19 +1,42 @@
 <script setup lang="ts">
 import { PageTitle } from '@/common/components'
+import { useAmsViewQuery } from '@/features/student/queries/use-ams-view.query/use-ams-view.query'
 import { studentHomeRoute } from '@/features/student/routes'
+import StudentDetailedAmsCard
+  from '@/features/student/views/StudentEducationAmsView/components/StudentDetailedAmsCard/StudentDetailedAmsCard.vue'
 import { useI18n } from 'vue-i18n'
 
+const { data: amss } = useAmsViewQuery(1, 1)
 const { t } = useI18n()
 
 const breadcrumbLinks = computed(() => [
   { text: t('student.navigation.tabs.home'), to: studentHomeRoute },
   { text: t('student.views.studentEducationAmsView.breadcrumb.current.title') }
 ])
+const displayWidget = computed(() => amss.value && amss.value.content && amss.value.content.length > 0)
 </script>
 
 <template>
   <PageTitle
     :title="t('student.views.studentEducationAmsView.title')"
     :breadcrumb-links="breadcrumbLinks"
   />
+  <div
+    v-if="displayWidget"
+    class="ams-widget-container__body"
```

Comment:
```suggestion
    class="ams-view-container__body"
```
On est sur un container, pas un widget ;)